modules/tinymce/src/plugins/link/test/ts/browser/UrlInputTest.ts

```diff
@@ -26,4 +38,15 @@ describe('browser.tinymce.plugins.link.UrlInputTest', () => {
     });
     TinyUiActions.closeDialog(editor);
   });
+  it('TINY-2884: insert url by keybord shortcut', async () => {
```

Comment:
This should have a new line between tests as mentioned earlier. There's also a typo in the label.
```suggestion

  it('TINY-2884: insert url by keyboard shortcut', async () => {
```